pkg/plugins/virtual/virtual_plugin.go

```diff
@@ -84,7 +84,17 @@ func (p *VirtualPlugin) Apply() error {
 	glog.Infof("virtual-plugin Apply(): desiredState=%v", p.DesireState.Spec)
 
 	if p.LoadVfioDriver == loading {
-		if err := utils.LoadKernelModule("vfio_pci"); err != nil {
+		// In virtual deployments of Kubernetes where the underlying virtualization platform does not support a virtualized iommu
+		// the VFIO PCI driver needs to be loaded with a special flag.
+		// This is the case for OpenStack deployments where the underlying virtualization platform is KVM.
+		kernel_args := []string{"enable_unsafe_noiommu_mode=1"}
```

Comment:
Is my understanding correct that vfio will from now on always be loaded with this arg? I'm unclear on the potential downsides of **always** loading it with this.